README.md

```diff
@@ -0,0 +1,50 @@
+# feature-selection-r
+An R package for feature selection with ML estimators
+
+
+### Contributors:
+|Name|GitHub|
+|----|-------------|
+|Ying Jing -Jacky- Ho |[jackyho112](https://github.com/jackyho112)|
+|Ryan Homer |[ryanhomer](https://github.com/ryanhomer)|
+|Derek Kruszewski |[dkruszew](https://github.com/dkruszew)|
+|Victor Cuspinera |[vcuspinera](https://github.com/vcuspinera)|
+
+### Overview:
+If you have face a database with so many features that while working it gets messy, a good idea is to select some of these features for your model. However, this is not a trivial question, and to help you with this task, we have created the `featureselection` package.
```

Comment:
I think we need to work on this some more but for the time being, it's a good placeholder.